audio/out/ao_coreaudio.c

```diff
@@ -476,29 +503,40 @@ static int hotplug_init(struct ao *ao)
         }
     }
 
-    return 0;
+    return true;
 
 coreaudio_error:
-    return -1;
+    return false;
 }
 
-static void hotplug_uninit(struct ao *ao)
+static bool unregister_hotplug_cb(struct ao *ao)
```

Comment:
is there a reason why this function returns a bool, but the return value isn't used anywhere?